server/include/lib_wrapper.hpp

```diff
@@ -0,0 +1,22 @@
+#ifndef INS_SERVER_INCLUDE_LIB_WRAPPER_HPP
+#define INS_SERVER_INCLUDE_LIB_WRAPPER_HPP
+
+#include <pistache/endpoint.h>
+#include <pistache/http.h>
+#include <pistache/router.h>
+
+using namespace Pistache::Http;
```

Comment:
[Don't use `using` in header files!](https://stackoverflow.com/a/5849668/6485320) :smiling_imp: